flashinfer/fused_moe/cute_dsl/blackwell_sm12x/moe_w4a16_fp4_helpers.py

```
@@ -197,6 +216,125 @@ def relu2_quantize_grouped_nvfp4_torch(
+        (num_groups, rows, cols // MX_SF_VEC_SIZE),
+        dtype=torch.uint8,
+        device=input_tensor.device,
+    )
+    for group_idx in range(num_groups):
+        valid_rows = int(row_counts[group_idx].item())
+        if valid_rows == 0:
+            continue
+        x = input_tensor[group_idx, :valid_rows].float()
+        blocked = x.view(valid_rows, cols // MX_SF_VEC_SIZE, MX_SF_VEC_SIZE)
+        block_max = blocked.abs().amax(dim=-1, keepdim=True)
+        rounded, byte = pow2_ceil_ue8m0_torch(block_max * _INV_FLOAT8_E4M3_MAX)
```

Ruff (0.16.0)

[warning] Unpacked variable `rounded` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)